wasm/src/graphics/webgl/bindings.rs

```diff
@@ -56,6 +56,19 @@ impl Gl2 {
             _ => unreachable!("unexpected return value from js function"),
         }
     }
+
+    pub fn link_program(&self, prog: u32) -> Result<(), ClientError> {
+        match unsafe { gl_link_program(prog as i32) } {
+            0 => Ok(()),
+            1 => Err(ClientError::WebGlError(
+                "program linkage failed, invalid program handle".to_string(),
+            )),
+            2 => Err(ClientError::WebGlError(
+                "program linkage failed, linker failed".to_string(),
+            )),
+            _ => unreachable!("unexpected return value from js function"),
```

Comment:
Why do you use unreachable instead of panic?

Reply:
`unreachable` is the same as `panic`. This marks, that this piece of code should be unreachable